Foundation/NSXMLNode.swift

```diff
@@ -229,7 +226,7 @@ public class NSXMLNode : NSObject, NSCopying {
         @abstract Returns an element, attribute, entity, or notation DTD node based on the full XML string.
     */
     public var kind: NSXMLNodeKind  {
-        guard _xmlNode != nil else { return .InvalidKind }
+//        guard _xmlNode != nil else { return .InvalidKind }
```

Comment:
I'm not sure what to do here. Pretty much nothing else in NSXMLNode can handle `_xmlNode` being `nil`. I suppose we could have asserting and non-asserting properties?